src/stdlib/typedarray.md

```diff
@@ -137,7 +142,28 @@ The TypedArray API works very much like JavaScript's \([MDN](https://developer.m
   function reverse(): this
   ```
   Reverses an array's values in place, modifying the array before returning it.
+* ```ts
+  function join(separator?: string): string;
+  ```
+  The join() method joins all elements of an array into a string. This method has the same algorithm as Array.prototype.join().
```

Comment:
instead `The <some-method>() method ...` pattern better follow previous convention. See `reduceRight` and etc when description start from "Calls ..." or `Joins all elements of an array into a string. `

Reply:
Yes, I know that. However, two documents (index.d.ts and website) need to be maintained for the same method, and the description styles are different, which is actually quite troublesome.

Anyway, I will modify it first